api/src/main/resources/messages.properties

```
@@ -92,6 +92,7 @@ error.rank.notPositiveInteger=Rank can only be a positive integer
 error.email.invalid =Invalid Email address
 error.activationkey.invalid =Invalid user activation Key
 error.usernameOrEmail.notNullOrBlank =Username Or email cannot be null or blank
+allergyapi.error.validateInputString=Other Reaction must not contain a number or symbol
```

```suggestion
.error.allergyapi.validateInputString=Other Reaction must not contain a number or symbol
```
This can be improved to follow code convention.